sumologic_opentelemetry/__init__.py

```diff
@@ -0,0 +1,7 @@
+"""sumologic-opentelemetry - An all-in-one package for python projects used to enable OpenTelemetry auto-instrumentation"""
+
+from . import env
+
+__version__ = '0.1.22'
```

Comment:
Please set version to correspond current opentelemetry-api/sdk packages version - in this case 1.9.1

Reply:
That would make it hard to add new features. So if we add automatic sources creation we would need to wait for opentelemetry-api/sdk to bump its version before publishing changes.
Is that ok?